core/frontend/.gitignore

```diff
@@ -0,0 +1,3 @@
+dist/
+node_modules/
+.eslintrc.js
```

Comment:
is this required? why?

Reply:
You mean the three of them or some specific one?

`dist/` for sure is not if one is only going to use `yarn serve` for development (thus not building it). `node_modules/` and `.eslintrc.js` I'm not really sure since the discovery on how eslnt find's it's packages.

I will take a look to see if any of those are not required.

Reply:
There's currently no .eslintrc.js file in core/frontend, is there?
Unless something is creating it, I don't think it is necessary to ignore it.

Reply:
Ok, just tested. 
The `.eslintrc.js` file does not need to be copied to that folder. `eslint-loader` is able to find the root one.
The `dist/` folder is there to allow the developer to try building the frontend locally (instead of serving it dinamically). Do you think it should be removed?
The `node_modules/` folder is indeed required, as one needs to `yarn install` on the frontend folder before serving it.

Current workflow is:

1. go to the root folder and install the packages need for testing, with the same command available on `.github/workflows/tests.yml` (as previously needed already)
2. go to the frontend folder and `yarn install`
3. `yarn serve` to begin testing/developing

Reply:
> There's currently no .eslintrc.js file in core/frontend, is there?
> Unless something is creating it, I don't think it is necessary to ignore it.

There's not. I just though before that for developing we would need to copy the eslint file to the frontend folder, but as the comment before states, it's not needed.